apps/astarte_realm_management/lib/astarte_realm_management/queries.ex

```diff
@@ -1777,18 +1777,9 @@ defmodule Astarte.RealmManagement.Queries do
   end
 
   def retrieve_realms!() do
-    statement = """
-    SELECT *
-    FROM #{CQLUtils.realm_name_to_keyspace_name("astarte", Config.astarte_instance_id!())}.realms
-    """
-
-    realms =
-      Xandra.Cluster.run(
-        :xandra,
-        &Xandra.execute!(&1, statement, %{}, consistency: :local_quorum)
-      )
+    keyspace = Realm.keyspace_name("astarte")
 
-    Enum.to_list(realms)
+    Repo.all(from(Realm), prefix: keyspace)
```

Comment:
The call to Repo.all is missing the `consistency` option.
As a small finesse, Ecto directly accepts the schema module without having to wrap it in a `from()`.

```elixir
Repo.all(Realm, prefix: keyspace, consistency: :local_quorum)
```